docs/api.rst

```diff
@@ -4833,7 +4833,7 @@ Decorators
 Cogs
 ~~~~
 
-.. autoclass:: ClientCog
+.. autoclass:: Cog
```

Comment:
Maybe this should stay as ClientCog to avoid breaking changes?

Reply:
I think an alias would be a better idea
But I think I should give out a deprecation warning (maybe in `__init_subclass__`) when someone uses it